### DISCUSSION TITLE
How often should the token be refreshed if I want to implement a third-party client

### DISCUSSION THREAD

**Discussion by benny856694** [pocketbase/pocketbase#1772] [Q&A] [answered]
The auth-with-password endpoint doesn't give details on when the token will expire

**Comment by ganigeorgiev** [comment, marked as answer]
The default auth token duration is 14 days. You can change it from the "Admin UI > Settings > Token options".

You can call `/auth-refresh` as many times as you want. PocketBase is completely stateless and the generated tokens are not stored in the database. All auth record tokens are automatically invalidated on password reset (_or on change of the common key part_).
